components/TargetDate.tsx

```diff
@@ -0,0 +1,28 @@
+import moment from 'moment'
+
+type Props = {
+    targetDate?: Date
+}
+
+/**
+ * Renders a datetime as a local date
+ * @param props
+ */
+export default function TargetDate(props: Props) {
+    if (props.targetDate) {
+        return (
+            <time dateTime={moment(props.targetDate).format('YYYY-MM-DD')}>
+                {formatTargetDate(props.targetDate)}
+            </time>
+        )
+    } else {
+        return '-'

```

Comment:
There's also a `<NullData />` component you can use here that renders this dash with consistent styling for all other null data (gray text, etc).